unlock-app/.gitignore

```diff
@@ -9,6 +9,9 @@
 # production
 /build
 
+# blog
+/src/static/blog.json
```

Comment:
Just a note: we don't want to accidentally check in the blog cache file, so I've included it in gitignore here.

Reply:
👍